### PR TITLE
Sectoral Activity Map - filter by activities

### PR DESCRIPTION
**Pull request by aabdaab**
This PR changes the behaviour of displaying data for sectoral indicators (`Waste`, `Energy` etc.). Before it was grouping the data by activity with the highest emission per province. Now we allow filtering by Activity on its own (dropdown `Activity`) - this gives us advantage of displaying all the data 
we have for provinces for specific activity. The map is coloured in a different way; it uses color mapping for specific range of emissions reflected in the legend.

image

### CODE REVIEW

**Comment by tsubik** [comment]
Is it correct that we only can see by one activity now? On designs, there was also "All" option, but that would complicate things as we would have to preserve previous behavior and add this one on top.

File: app/javascript/app/pages/national-context/sectoral-activity/selectors/sectoral-activity-constants.js [hunk 1]
```diff
@@ -15,11 +16,17 @@ export const SECTION_COLORS = {
   ENERGY: '#2EC9DF',
   FORESTRY: '#FCA683',
   INDUSTRIAL_PROCESS_AND_PRODUCT_USE: '#FFC735',
-  'no-data': '#ffffff'
+  'no-data': '#FAFAFA'
```
Comment (tsubik) [inline app/javascript/app/pages/national-context/sectoral-activity/selectors/sectoral-activity-constants.js]: My :eye: don't see a difference between this and first bucket white color. But as a :mens: I could be just color blind.

**Comment by tsubik** [comment]
I see that Primary Source of Emission preserves the old behavior, so we could add "All" if they want it in the next iteration maybe.

Anyway, beyond similar color for no-data and first bucket, other things and code LGTM! :+1:

File: app/javascript/app/pages/national-context/sectoral-activity/selectors/sectoral-activity-constants.js (hunk 1 above)
Comment (aabdaab) [reply app/javascript/app/pages/national-context/sectoral-activity/selectors/sectoral-activity-constants.js]: You're right, that is a bit unclear, for a 🚺 as well 😂 I'm gonna check with Fausto what does he think is the best color for no-data.

**Pull request merged**